src/www/wmnds-website.js

```diff
@@ -19,4 +20,7 @@ const icons = () => {
   };
 };
 
-window.addEventListener('DOMContentLoaded', (forEachPolyfill, icons(), colorPalettes(), headerJs(), highlightJS));
+window.addEventListener(
+  'DOMContentLoaded',
+  (forEachPolyfill, icons(), colorPalettes(), headerJs(), cookies(), highlightJS)
+);
```

Comment:
You will need to do a fetch/pull on master and merge master into your cookie branch as there is a conflict (your version of wmnds-website.js is out of date compared with live version).

So just make sure you have the latest changes in there as well as your own.

Reply:
All good now! 👍